src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -48,4 +83,47 @@ export class DashboardComponent {
 
     this.router.navigateByUrl(url);
   }
+
+  /**
+   * Retrieves an array of learningObjects to populate the draft and released list of learning objects
+   * @param filters
+   * @param query
+   */
+  async getLearningObjects(filters?: any, text?: string): Promise<LearningObject[]> {
```

Comment:
Can we replace this function with a `getReleasedLearningObjects` function?

Reply:
Resolved with rename function